chrome-launcher.ts

```diff
@@ -79,7 +79,7 @@ export async function launch(opts: Options = {}): Promise<LaunchedChrome> {
     return instance.kill();
   };
 
-  return {pid: instance.pid!, port: instance.port!, kill};
+  return {pid: instance.pid!, port: instance.port!, kill, chromeProcess: instance.chrome};
```

Comment:
can you add a test to assert this chromeProcess prop exists on the return value.. and that it's an instanceof childProcess?

Reply:
you want an assertion in the codebase or a unit test?

Reply:
hah good question. a unit test. :)  glanced quick and we don't have anything which asserts the shape of the return object API, but that generally makes sense to me.

Reply:
I added a unit test for the launch helper function return signature.

also, there was no .gitignore file, so node_modules is committed, fyi.

Reply:
I am going to assume that you want me to ignore node_modules...lmk

Reply:
Looks like there are a lot of changes to .js and .d.ts files, should we add those to .gitignore